packages/@aws-cdk/aws-ivs/README.md

```diff
@@ -9,8 +9,62 @@
 ---
 <!--END STABILITY BANNER-->
 
+Amazon Interactive Video Service (Amazon IVS) is a managed live streaming
+solution that is quick and easy to set up, and ideal for creating interactive
+video experiences. Send your live streams to Amazon IVS using streaming software
+and the service does everything you need to make low-latency live video
+available to any viewer around the world, letting you focus on building
+interactive experiences alongside the live video. You can easily customize and
+enhance the audience experience through the Amazon IVS player SDK and timed
+metadata APIs, allowing you to build a more valuable relationship with your
+viewers on your own websites and applications.
+
 This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk) project.
 
+### Channels
+
+An Amazon IVS channel stores configuration information related to your live
+stream. You first create a channel and then contribute video to it using the
+channel’s stream key to start your live stream.
+
+You can create a channel
+
+```ts
+let myChannel = new ivs.Channel(this, 'Channel');
+```
+
+You can reference an existing channel, for example, if you need to create a
+stream key for an existing channel
+
+```ts
+let myChannel = ivs.Channel.fromChannelArn(this, 'Channel', myChannelArn)
+```
+
+You can create stream keys
+
+```ts
+new ivs.StreamKey(this, 'StreamKey', {
+  channel: myChannel,
+});
+```
+
+### Private Streams
+
+Amazon Interactive Video Service (IVS) offers customers the ability to create
+private channels, allowing customers to restrict their streams by channel or
+viewer. Customers control access to video playback by enabling playback
+authorization on channels and generating signed JSON Web Tokens (JWTs) for
+authorized playback requests.
+
+A playback token is a JWT that the Amazon IVS customer signs (with a playback
+authorization key) and includes with every playback request for a channel that
+has playback authorization enabled.
+
+In order for Amazon IVS to validate the token, the customer needs to upload
+the public key that corresponds to the private key they used to sign the token.
+
 ```ts
-import ivs = require('@aws-cdk/aws-ivs');
+new ivs.PlaybackKeyPair(this, 'PlaybackKeyPair', {
+  publicKeyMaterial: myPublicKeyPemString,
+});
 ```
```

Comment:
similar questions to the StreamKey... these feels like resources that we would need to create as operations on a channel?

thinking from a user's perspective... when would they want to create this resource? it feels like this should also be an API on the `Channel` construct

Reply:
Playback keys aren't tied to a channel

Reply:
okay, let's step back to the user and how they leverage these in that case. when does a user need to create a playback key pair? 

it would help to understand the context in which one needs to be created/used to clarify why a separate construct is required.

Reply:
Yeah, basically they'd need to create one if any of the channels in the account/region need authorisation.